Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/StudentControllerTest.java

```diff
@@ -290,6 +292,24 @@ void Given10StudentsAnd4Admins0_1_2_3_WhenGetAdminsWithSkip2Size2_ShouldRespondA
         assertEquals("admin3", students.get(1).name);
     }
 
+    @Test
+    void GivenStudentSignedUp_WhenDeleteTheStudent_ShouldSucceed() throws Exception {
+        StudentView student = signUpAndGetResponseBody(this.student);
+
+        deleteStudent(student.id);
+
+        assertTrue(studentRepository.findStudentById(student.id).isEmpty());
+    }
+
+    @Test
+    void WhenDeleteTheStudent_ShouldSucceed() throws Exception {
+        int studentId = 123;
+
+        deleteStudent(studentId);
```

Comment:
Because you do not have ```given``` , you don't need a line to separate two lines.
Change it like below.
```Java
int studentId = 123;
deleteStudent(studentId);
```